testutil/promrated/promrated.go

```diff
@@ -74,7 +74,7 @@ func Run(ctx context.Context, config Config) error {
 	}
 }
 
-// report the validator effectiveness metrics for prometheus
+// report the validator effectiveness metrics for prometheus.
```

Comment:
my golangci-lint fixed this automatically... weird.

Reply:
something's wrong, you should check it